src/mail/view/MailViewerUtils.ts

```diff
@@ -191,6 +196,12 @@ export function mailViewerMoreActions(viewModel: MailViewerViewModel, showReadBu
 		})
 	}
 
+	moreButtons.push({
```

Comment:
We agreed to add only the shortcut and don't add an extra option inside more actions

Reply:
Okay, I will remove the button from the more menu.

Reply:
What was the explanation for this? We have a button for showing email headers inside the "More" list, so not having one for the email source feels a bit inconsistent.